client/src/components/SignInAlert/SignInAlert.js

```diff
@@ -0,0 +1,32 @@
+import React from "react";
+import { Grid, Col, Row, Jumbotron, Button } from 'react-bootstrap';
+import './SignInAlert.css';
+
+const SignInAlert = () => (
+    <Grid className='noMatch'>
+        <Row>
+            <Col xs={12}>
+                <Jumbotron className='jumbo'>
+                    <h1>You Hop'd To Far</h1>
+                    <h2>
```

Comment:
"to" should be "too"
So it should be "You Hop'd Too Far"

Reply:
But besides that it looks good. Good job!

Reply:
Nice catch! I'll switch it, push, and merge